Differ.xcodeproj/project.pbxproj

```diff
@@ -377,7 +377,7 @@
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_VERSION = 5.0;
-				TARGETED_DEVICE_FAMILY = "1,2,4";
+				TARGETED_DEVICE_FAMILY = "1,2,3,4";
```

Comment:
I don't see the warning about `TARGETED_DEVICE_FAMILY` using Xcode 10.2. This build setting is set correctly via the `xcconfig` files included with the project (see `Universal-Framework-Target.xcconfig`). Could you provide more detail about this problem in a separate issue/PR, please?